src/calendar-list/index.tsx

```diff
@@ -239,7 +242,7 @@ const CalendarList = (props: CalendarListProps, ref: any) => {
       return (
         <CalendarHeader
           {...headerProps}
-          testID={STATIC_HEADER}
+          testID={`${testID}.header`}
```

Comment:
This id might cause duplications, as it is an additional header on top of the Calendar header (CalendarListItem). We should consider calling it `staticHeader` or have a default value to the `testID` prop as "calendarList".

Reply:
Fixed